src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -81,6 +82,8 @@
 public class LoginAPI implements Authn {
 
   private static final String TABLE_NAME_CREDENTIALS = "auth_credentials";
+  private static final String CONTENT_TYPE_HEADER = "Content-Type";
```

Comment:
as for me it's better to use `javax.ws.rs.core.HttpHeaders` constants instead of defining your own

Reply:
converted to `javax.ws.rs.core.HttpHeaders`